### PR TITLE
 Fixing the necessary script changes for terraform version 13

### PR DESCRIPTION
**Pull request by RagniGarg97**
The code works for the following versions.
Terraform v0.13.0
+ provider registry.terraform.io/hashicorp/ibm v1.21.2
+ provider registry.terraform.io/hashicorp/null v2.1.2

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/31#" title="Author self-approved">RagniGarg97</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fkubetest2-plugins).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/kubetest2-plugins/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by Rajalakshmi-Girish** [comment]
@RagniGarg97 does this even work for the previous version of Terraform?
Else will have to do all the required Terraform Upgrade changes in one go.
- Change Terraform version to 0.13 in `all-in-one` image
- If newly added `all-in-one` image has an updated version tag, the conformance jobs must be changed to use latest version of all-in-one.

**Comment by Rajalakshmi-Girish** [comment]
@mkumatag Can you please take a look?

**Review by mkumatag — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/31#pullrequestreview-636602736" title="LGTM">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/31#" title="Author self-approved">RagniGarg97</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fkubetest2-plugins).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/kubetest2-plugins/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
